Documentation/Diagnostics/PH2080.md

```diff
@@ -12,7 +12,7 @@
 
 ## Introduction
 
-Avoid hardcoded absolute paths, as they are bound the change without notice. 
+Avoid hardcoded absolute paths, as they are bound the change without notice. For performance reasons, only the first 1000 charcters of a string literal are inspected.
```

Comment:
Suggest adding note that this analyzer works on Windows-style paths only.

Reply:
Updated documentation